src/test/java/uk/gov/digital/ho/proving/income/audit/AuditArchiveServiceIT.java

```diff
@@ -30,6 +31,7 @@
 public class AuditArchiveServiceIT {
 
     @Autowired
+    @Qualifier("createAuditRestTemplate")
```

Comment:
You don't have to change it - this works fine - but maybe for your learning you could see if removing this and renaming the variable to auditRestTemplate also works. I'd be keen to hear.

Reply:
The tests failed once I'd renamed the template method and removed the qualifier so I've left it in. However I took the opportunity to rename the rest template methods anyway.